package.json

```diff
@@ -84,9 +84,9 @@
     "postcss-loader": "2.1.1",
     "postcss-remove-classes": "1.0.4",
     "raw-loader": "1.0.0-beta.0",
-    "react": "16.2.0",
+    "react": "16.3.2",
     "react-dev-utils": "5.0.0",
-    "react-dom": "16.2.0",
+    "react-dom": "16.3.2",
     "react-test-renderer": "16.2.0",
```

Comment:
There's a 16.3.2 of this as well. I'd edit inline but we need an updated lock file too.